Source/Expectation+Ext.swift

```diff
@@ -1,12 +1,19 @@
 import Nimble
 
 extension Expectation {
+    #if swift(>=4.1)
```

Comment:
Are we sure this shouldn't be `4.2` instead? Xcode 9 shows Swift version is 4.1.2, but Xcode 10 beta shows 4.2. I got a validation error while pushing to CocoaPods because of this, which I'm fine circumventing as long as this is correct 👍

Reply:
Nah, never mind you're right. Hmm, I guess the validation won't be fixed until there's a new Nimble release with the public constructor. I'll wait on that.

In the meantime, anyone can point their Podfile to this repo's `master` branch to use this change right away 👍

Reply:
Hmm, then why is CI working but my local Xcode 9 isn't? https://travis-ci.org/RxSwiftCommunity/RxNimble/jobs/391941472